### PR TITLE
 fix(sdk): Correct "subscribe" typos

### PR DESCRIPTION
**Pull request by jhermsmeier**
Change-Type: patch

### CODE REVIEW

File: dictionary [hunk 1]
```diff
@@ -3,3 +3,4 @@ aknowledge->acknowledge
 seleted->selected
 reming->remind
 locl->local
+subsribe->subscribe
```
Comment (lurch) [inline dictionary]: Could you add `unsubsribe` too? ;)

Comment (jhermsmeier) [reply dictionary]: Sure thing, added!

File: lib/shared/sdk/index.js [hunk 2]
```diff
@@ -192,28 +192,28 @@ SDK.Scanner = class Scanner extends EventEmitter {
   }
 
   /**
-   * @summary Subsribe to an adapter
+   * @summary subscribe to an adapter
```
Comment (lurch) [inline lib/shared/sdk/index.js]: Minor nit: should there be a capital 'S' on 'subscribe' here?

Comment (jhermsmeier) [reply lib/shared/sdk/index.js]: Yup, fixed! The pleasures of global find & replace tricked me here.

Comment (lurch) [reply lib/shared/sdk/index.js]: Sometimes those find'n'replace dialogs have a handy "match exact case only" tickbox :stuck_out_tongue_winking_eye:

**Pull request merged**